### ISSUE TITLE
open-babel: error: 'std::tr1' has not been declared

### ISSUE THREAD

**Issue by rrchaudhari** [Linuxbrew/homebrew-core#217]
### Bug reports:

Not able to install open-babel on CentOS machine using brew. 
Here is the gist file. 

https://gist.github.com/anonymous/592947d7cde6b35e7448c25d25be5f65

Thank you.

**Comment by rwhogg** [comment]
Hey @rrchaudhari, you accidentally put the bug-creation link instead of the gist file. Can you copy the url from `brew gist-logs` please?

In the meantime, I will build a bottle for open-babel on Travis and see if there's a problem there.

Thanks!

**Comment by rwhogg** [comment]
See https://github.com/Linuxbrew/homebrew-core/pull/219

**Comment by rrchaudhari** [comment]
Sorry, I have updated the link.

**Comment by rrchaudhari** [comment]
By the way I also had issues installing julia using linuxbrew. I should actually learn how to create bottles instead. Thank you @rwhogg for your help.

**Comment by sjackman** [comment]
https://gist.github.com/anonymous/592947d7cde6b35e7448c25d25be5f65#file-02-make-L45
```
cd /tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/build/src/formats/libinchi && /usr/local/brew/bin/gcc-5  -DBUILD_LINK_AS_DLL -DHAVE_EIGEN -DHAVE_EIGEN3 -DHAVE_LIBZ -DHAVE_SHARED_POINTER -DUSING_DYNAMIC_LIBS -D_USRDLL -I/usr/local/brew/include -I/tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/build/include -I/tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/data -I/tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/include -I/usr/local/brew/include/eigen3 -I/tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/include/inchi  -Os -w -pipe -march=core2  -DNDEBUG -fPIC   -DOB_MODULE_PATH="\"/usr/local/brew/Cellar/open-babel/2.3.2/lib/openbabel/2.3.2\"" -o CMakeFiles/inchi.dir/ichimak2.o   -c /tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/src/formats/libinchi/ichimak2.c
In file included from /tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/include/openbabel/alias.h:19:0,
                 from /tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/src/alias.cpp:17:
/tmp/open-babel-20160505-22141-1jcorhv/openbabel-2.3.2/include/openbabel/shared_ptr.h:34:16: error: 'std::tr1' has not been declared
     using std::tr1::shared_ptr;
```
